### PR TITLE
9 formulas functions created and tested

### PR DESCRIPTION
**Pull request by i-abdul-raheem**

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -1,47 +1,104 @@
...
+) => {
+  const portionSum = sandportion + cementportion + bajarportion;
+  return (sandportion / portionSum) * dryMaterial;
+};
+
+const calculateCrushCementCft = (
+  sandportion: number,
+  cementportion: number,
+  bajarportion: number,
+  dryMaterial: any
```
Comment (i-abdul-raheem) [inline src/utils/utils.ts]: Change dryMaterial type to a valid type

File: src/utils/utils.ts [hunk 2]
```diff
@@ -1,47 +1,104 @@
...
+const crushDryMaterial = (bajarucft: number) => {
+  const thirtyPercent = (30 / 100) * bajarucft;
+  return bajarucft + thirtyPercent;
+};
+
+const calculateCrushSandCft = (
+  sandportion: number,
+  cementportion: number,
+  bajarportion: number,
+  dryMaterial: any
```
Comment (i-abdul-raheem) [inline src/utils/utils.ts]: Change dryMaterial type to a valid type

File: src/utils/utils.ts [hunk 3]
```diff
@@ -1,47 +1,104 @@
...
+) => {
+  const portionSum = sandportion + cementportion + bajarportion;
+  return (sandportion / portionSum) * dryMaterial;
+};
+
+const calculateBajarCementCft = (
+  sandportion: number,
+  cementportion: number,
+  bajarportion: number,
+  dryMaterial: any
```
Comment (i-abdul-raheem) [inline src/utils/utils.ts]: Change dryMaterial type to a valid type

File: src/utils/utils.ts [hunk 4]
```diff
@@ -1,47 +1,104 @@
...
+const bajarDryMaterial = (bajarcft: number) => {
+  const thirtyPercent = (30 / 100) * bajarcft;
+  return bajarcft + thirtyPercent;
+};
+
+const calculateBajarSandCft = (
+  sandportion: number,
+  cementportion: number,
+  bajarportion: number,
+  dryMaterial: any
```
Comment (i-abdul-raheem) [inline src/utils/utils.ts]: Change dryMaterial type to a valid type

**Review by console-dot — CHANGES_REQUESTED** [review]
Requested changes. Please fix them

**Review by console-dot — APPROVED** [review]

**Pull request merged**